### PR TITLE
revise the configuration for urls

### PR DESCRIPTION
**Pull request by wy65701436**
1, doesn't return error for registry url
2, give the default value for jobservice

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by chlins — APPROVED** [review]
lgtm

File: src/lib/config/systemconfig.go [hunk 1]
```diff
@@ -47,17 +47,21 @@ func TokenPrivateKeyPath() string {
 }
 
 // RegistryURL ...
-func RegistryURL() (string, error) {
+func RegistryURL() string {
 	url := os.Getenv("REGISTRY_URL")
```
Comment (stonezdj) [inline src/lib/config/systemconfig.go:51]: Suggest to use:
```
    return DefaultMgr().Get(backgroundCtx, common.RegistryURL).GetString()
```

File: src/lib/config/metadata/metadatalist.go [hunk 2]
```diff
@@ -181,6 +184,7 @@ var (
 		{Name: common.CacheExpireHours, Scope: SystemScope, Group: BasicGroup, EnvKey: "CACHE_EXPIRE_HOURS", DefaultValue: "24", ItemType: &IntType{}, Editable: false, Description: `The expire hours for cache`},
 
 		{Name: common.GDPRDeleteUser, Scope: SystemScope, Group: GDPRGroup, EnvKey: "GDPR_DELETE_USER", DefaultValue: "false", ItemType: &BoolType{}, Editable: false, Description: `The flag indicates if a user should be deleted compliant with GDPR.`},
+		{Name: common.CacheExpireHours, Scope: SystemScope, Group: SystemScope, EnvKey: "GC_TIME_WINDOW_HOURS", DefaultValue: "2", ItemType: &Int64Type{}, Editable: false, Description: `The time window hours for garbage collection`},
```
Comment (chlins) [inline src/lib/config/metadata/metadatalist.go:187]: should be `common.GCTimeWindowHours`?

Comment (wy65701436) [reply src/lib/config/metadata/metadatalist.go]: updated

**Review by stonezdj — APPROVED** [review]

**Pull request closed without merging**